cypress/support/elements/common/PrimaryWorkspace.js

```diff
@@ -36,28 +36,40 @@ class RightNav{
         return this.testDocumentWithTitle(tab, subTab, title, 'not.contain');
     }
 
-    getRightNavTabs(){
+    getPrimaryWorkspaceTabs(){
         return cy.get('.right-nav .tabs');
     }
 
-    closeRightNavTabs(){
+    closePrimaryWorkspaceTabs(){
```

Comment:
Is this to collapse the workspace? If it is to close the resource panel, it should probably go to the resource panel elements file.